rviz_ogre_vendor/package.xml

```diff
@@ -30,6 +30,7 @@
   <depend>libxaw</depend>
   <depend>libxrandr</depend>
   <depend>opengl</depend>
+  <depend>pugixml-dev</depend>
```

Comment:
Is this a new dependency of OGRE 1.12.12?  Can we avoid this dependency?  If not, we'll have to figure out a way to package and/or vendor it for Windows and macOS.